.travis.yml

```diff
@@ -10,10 +10,14 @@ cache:
   - $HOME/maven
 env:
   matrix:
-  - MAVEN_VERSION=3.5.4 TEST=core
-  - MAVEN_VERSION=3.5.4 TEST=python-validator
-  - MAVEN_VERSION=3.5.4 TEST=sanity-checks
-  - MAVEN_VERSION=3.5.4 TEST=end-to-end
+  - 
+  =3.5.4 TEST=core
+  - 
+  =3.5.4 TEST=python-validator
+  - 
+  =3.5.4 TEST=sanity-checks
+  - 
+  =3.5.4 TEST=end-to-end
```

Comment:
@rahul799 i think travis is not working because of these changes? If you revert them it might work

Reply:
or did u edit it after it stopped working?

Reply:
Hope Travis won't fail this time. :crossed_fingers: